packages/office-ui-fabric-react/src/components/Link/examples/Link.Basic.Example.tsx

```diff
@@ -1,28 +1,21 @@
 import * as React from 'react';
 import { Link } from 'office-ui-fabric-react/lib/Link';
-import * as styles from './Link.Example.scss';
 
-export class LinkBasicExample extends React.Component<any, any> {
-  public render(): JSX.Element {
-    return (
-      <div className={styles.linkExample}>
-        <span>When a link has an href, </span>
-        <Link href="http://dev.office.com/fabric/components/link">it renders as an anchor tag.</Link>
-        <span> Without an href, </span>
-        <Link>the link is rendered as a button</Link>.<span> You can also use the disabled attribute to create a </span>
+export const LinkBasicExample: React.FunctionComponent = () => {
+  return (
+    <div>
+      <p>
+        When a link has an href, <Link href="http://dev.office.com/fabric/components/link">it renders as an anchor tag.</Link> Without an
+        href, <Link>the link is rendered as a button</Link>. You can also use the disabled attribute to create a{' '}
         <Link disabled={true} href="http://dev.office.com/fabric/components/link">
           disabled link.
-        </Link>
-        <span>
-          {' '}
-          Example of a Link with <Link href="http://dev.office.com/fabric/components/link">mixed link content that wraps.</Link>
-        </span>
-        <br />
-        <br />
+        </Link>{' '}
```

Comment:
I'm guessing the `{' '}` at the end was added automatically by prettier here?

Reply:
It's actually just a leftover--there used to be an additional Link in this example, but I removed it because it didn't seem to be adding anything to the example.